### PR TITLE
fix(labResult): value display

### LINKED ISSUE #101: Afficher tout type de valeur dans les résultats de biologie
## Description
Pour le moment, seulement les attributs de type ValueQuantity sont affichés correctement dans les résultats de bio, alors qu'il existe des résultats de type ValueString, ValueBoolean, ValueRatio etc.. 

En particulier, on a besoin de résultats de type ValueString pour un type de résultats spécifique.

### PR DESCRIPTION
**Pull request by tevariou**
## Fixes

Fixes #101 and #102  by @Peltier10 and @elsiehoffet-94

### CODE REVIEW

File: src/components/Patient/PatientLaboTable/PatientLaboTable.tsx [hunk 1]
```diff
@@ -27,17 +28,30 @@ const PatientLaboTable = ({ patient }: PatientLaboTableProps) => {
...
+    if (res.valueString) return res.valueString
+    if (res.valueTime) return res.valueTime
+    if (res.valueDateTime) return res.valueDateTime
+    if (res.valueInteger) return res.valueInteger.toString()
+    if (res.valueRange?.low || res.valueRange?.high)
+      return `${res.valueRange.low?.value} ${res.valueRange.low?.unit} - ${res.valueRange.high?.value} ${res.valueRange.high?.unit}`
+    if (res.valuePeriod) return `${res.valuePeriod.start} - ${res.valuePeriod.end}`
+    if (res.valueRatio?.numerator || res.valueRatio?.denominator)
+      return `${res.valueRatio.numerator?.value} ${res.valueRatio.numerator?.unit} / ${res.valueRatio.denominator?.value} ${res.valueRatio.denominator?.unit}`
+    if (res.valueCodeableConcept?.text) return res.valueCodeableConcept.text
```
Comment (elsiehoffet-94) [inline src/components/Patient/PatientLaboTable/PatientLaboTable.tsx:43]: @nriss maybe we can integrate this to Cohort360 IG. We can add a MustSupport on the text attribute if valueCodeableConcept is filled, to make sure something will appear on the app.

**Review by elsiehoffet-94 — APPROVED** [review]
Great, I'm curious to test it on all types of values 👓

**Review by Peltier10 — APPROVED** [review]
Looks ok 😄

File: src/components/Patient/PatientLaboTable/PatientLaboTable.tsx (hunk 1 above)
Comment (nriss) [reply src/components/Patient/PatientLaboTable/PatientLaboTable.tsx:43]: Yep! 
Also discussed there https://github.com/arkhn/Cohort360/issues/95 about which attribute to show, I suggested `display > text > code` depending on which attribute is filled. The order `text > display > code` seems also great

image

**Review by MiskoG — APPROVED** [review]
👍

**Review by simonvadee — APPROVED** [review]

**Pull request merged**